src/api/RestAPI.ts

```diff
@@ -325,8 +326,12 @@ export class RestAPI {
                 (async function repeater(): Promise<A> {
                     const result = await promise();
 
+                    if (cancelCondition && cancelCondition(result)) {
```

Comment:
hmm, just wondering, is it possible to write `cancelCondition?.(result)` in the conditional here?

Reply:
We are not using typescript 3.7 on this project and I was afraid to break something by doing the update 🤔